IaC/AKS/README.md

```diff
@@ -410,42 +420,18 @@ envsubst < clusterissuer.yaml | kubectl apply -f -
 
 ## Deploy NGSA with Helm
 
-A helm chart is included for the reference application ([NGSA](https://github.com/retaildevcrews/ngsa))
-
-A file called helm-config.yaml with the following contents that needs be to edited to fit the environment being deployed in. The file looks like this
-
-```yaml
-
-# Default values for NGSA.
-# This is a YAML-formatted file.
-# Declare variables to be passed into your templates.
-image:
-  repository: retaildevcrew
-  name: ngsa
-  tag: beta
-
-ingress:
-  hosts:
-    - %%APP_ENDPOINT%%
-  paths:
-    - /
-
-app:
-  args: []
-
-```
-
-Replace the values in the file surrounded by `%%` with the proper environment variables
+A helm chart is included for the reference application ([NGSA](https://github.com/retaildevcrews/ngsa)).
```

Comment:
does this helm chart have the sidecar setup info already?

Reply:
It will need to be added, we thought about adding that in a separate PR, but will discuss with AK to see if this PR works.

Reply:
create an issue and address in the next PR is fine

Reply:
References #107